web/src/components/pages/contribution/speak/recording-pill.css

```
@@ -49,4 +49,4 @@
     & .placeholder {
         width: 55px;
     }
-}
+}
```

```suggestion
}

```
Can you add the blank line to the end of this file so this file does not show us a change in this PR anymore? Once that is done I'll merge this. Thanks!